### ISSUE TITLE
Only show cumulative cases from today backwards. Will help to scale the cluster in the future.

### ISSUE THREAD

**Issue by sibeshkar** [someshkar/covid19india-cluster#141]
Selecting today's date in date picker show only today's cases. Selecting yesterday's date shows today + yesterday's cases. And so on. This is urgent.

**Comment by sampathBlam** [comment]
@sibeshkar : Please clarify. Currently the application is behaving the other way right? 
Current behavior:
* Selecting today in the date picker shows all cases till today. 
* Selecting yesterday in the date picker shows all cases till yesterday. 

Expected behavior:
* Selecting today in the date picker shows cases that are reported today alone. 
* Selecting yesterday in the date picker shows cases that are reported yesterday and today.
* Selecting day before yesterday in the date picker shows cases that are reported for the last three days. 

So when you select any date, the cluster will hold data for those cases that were reported from that day till today. 

Is this fine? I believe the current implementation seems more relevant and easier to understand than the proposed change.

**Comment by sibeshkar** [comment]
Yes you're right, but it's getting quite laggy and will be difficult to study as cases balloon (touchwood) into thousands, and seems to be crashing on mobile. Hence this change. Going back and clicking on 23rd should show you all patients, if your computer can deal with it

**Comment by sampathBlam** [comment]
Okay. I will work on it. Lets see how it pans out. We always have the option of backing out to the current strategy if we can get the data processed remotely in an efficient manner.

**Comment by sibeshkar** [comment]
Thank you :smiley:

**Comment by sampathBlam** [comment]
@sibeshkar : I think we need to rely on API changes from https://api.rootnet.in.
It seems that whenever we send a request to api.rootnet.in with a date value in the url (For eg: https://api.rootnet.in/covid19-in/unofficial/covid19india.org/patientdb/2020-03-24), internally api.rootnet.in calls https://covid19-data.rootnet.in/covid19india.org/patientdb-${date}.json. And that returns all cases reported till that date. 

Maybe we can just get all data till the current day from api.rootnet.in, and then filter on our side in the frontend. Your thoughts please?

**Comment by sibeshkar** [comment]
Simple way around this - if date X is selected than show all cases reported on X and after, retrieved from the API. So if 23rd is selected, we show all cases. If today is selected, it just shows the ones reported today.

**Comment by sampathBlam** [comment]
If a person A gets infected today, and we show only today's cases when the user selects today in the Date section, then we would not be able to show from whom person A got infected. 
This takes a huge toll on the utility of the cluster itself. The cluster is of very less use without the information about the source of infection for a patient. Isn't it ?

**Comment by saikatHi6** [comment]
I have a suggestion for the same. We can display only today's patines and can give an option in detail page from whom the person got infected or if somebody clicked on node it will show the patient effected by whom, means detail graph.

**Comment by sibeshkar** [comment]
Let's just keep including patients from whom disease has been contracted, regardless of date.
